integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/resources/webhook/site/WebhookSiteRequest.java

```diff
@@ -1,28 +1,18 @@
 package com.redhat.service.smartevents.integration.tests.resources.webhook.site;
 
-import java.util.Date;
+import java.time.LocalDateTime;
 import java.util.List;
 import java.util.Map;
 
-import com.fasterxml.jackson.annotation.JsonFormat;
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
-import com.fasterxml.jackson.annotation.JsonProperty;
-
-@JsonIgnoreProperties(ignoreUnknown = true)
 public class WebhookSiteRequest {
 
-    @JsonProperty
     private String uuid;
 
-    @JsonProperty
     private Map<String, List<String>> headers;
 
-    @JsonProperty
     private String content;
 
-    @JsonProperty("created_at")
-    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd HH:mm:ss")
-    private Date createdAt;
+    private LocalDateTime createdAt;
```

Comment:
what's the reason behind to change from Date to LocalDateTime? I know that TNB is using LocalDateTime as well, there and maybe it might be easier to set it to the same date type, but just curious.

Reply:
yes, I have aligned it to TNB for simplicity